nerfstudio/data/dataparsers/base_dataparser.py

```diff
@@ -60,6 +60,17 @@ class DataparserOutputs:
     """Scene box of dataset. Used to bound the scene or provide the scene scale depending on model."""
     mask_filenames: Optional[List[Path]] = None
     """Filenames for any masks that are required"""
+    depths: Optional[List[torch.Tensor]] = None
+    """Monocular depth."""
+    normals: Optional[List[torch.Tensor]] = None
+    """Monocular normal."""
+    additional_inputs: Dict[str, Any] = to_immutable_dict({})
```

Comment:
Is `additional_inputs` necessary. It is currently being used in the following form:
```
def get_x_masks(image_idx, x):
    return {"x": x[image_idx]}

additional_inputs_dict["foreground_masks"] = {
                "func": get_x_masks,
                "kwargs": {"x": foreground_mask_images},
            }
```
It seems like this masking can be done elsewhere.

Reply:
Can you have a config like neus that contains an arguments for "use_monocula_cues" that will only use the values if it is set to true, otherwise ignore them?